mokapot/mokapot.py

```diff
@@ -82,9 +96,30 @@ def main():
                 dataset.add_proteins(proteins)
 
     # Define a model:
+    model = None
     if config.load_models:
         model = [load_model(model_file) for model_file in config.load_models]
-    else:
+    elif enabled_plugins:
+        plugin_models = {}
+        for plugin_name, plugin in enabled_plugins.items():
+            model = plugin.get_model(config)
+            if model is not None:
+                logging.debug(f"Loaded model for {plugin_name}")
+                plugin_models[plugin_name] = model
+
+        if len(plugin_models) == 0:
+            msg = "No models were defined by plugins. Using default model."
+            logging.debug(msg)
```

Comment:
**nitpik:** I'd put `msg` in the `loggin.debug()` statement instead of creating a variable.

Reply:
same as below, it gets to +80 columns ... I am not sure what the preffered style for this proejct is to solve this kind of issues.